### PR TITLE
Added GitHub Actions, fixed PHP 8.4 deprecation

### PR DESCRIPTION
**Pull request by davidbyoung**

### CODE REVIEW

**Comment by davidbyoung** [comment]
@erusev I've successfully gotten the CI tests to pass across PHP versions 7.1 - 8.4.  Should be ready for review.

**Comment by davidbyoung** [comment]
@erusev FYI this is ready for review

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
-vendor/
+.idea/
```
[the remark this answers is not in the document]
Comment (davidbyoung) [reply .gitignore]: Removed in the latest commit

File: composer.json [hunk 2]
```diff
@@ -9,20 +9,23 @@
...
             "name": "Emanuil Rusev",
             "email": "hello@erusev.com",
-            "homepage": "http://erusev.com"
+            "homepage": "https://erusev.com"
         }
     ],
     "require": {
-        "erusev/parsedown": "^1.8.0|^1.8.0-beta-4",
-        "php": ">=5.3.6",
+        "erusev/parsedown": "1.8.x-dev",
```
[the remark this answers is not in the document]
Comment (davidbyoung) [reply composer.json]: Updated to dev-master because that contains the latest commits.  Let me know if there's a specific version I should be pointing to.

**Comment by robsontenorio** [comment]
@erusev could you please, release a new tag with this fix?

**Comment by davidbyoung** [comment]
@erusev Can you tag 0.8.2?

**Pull request merged**